src/components/my_reservations/MyReservations.jsx

```diff
@@ -0,0 +1,99 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import deleteReservation from '../../redux/my_reservations/actions/deleteReservation';
+import fetchMyReservations from '../../redux/my_reservations/actions/fetchMyReservations';
+import { selectUserId } from '../../redux/user/userSlice.js';
+import { formatDistanceToNow } from 'date-fns';
+
+function MyReservations() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const MyReservations= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

Reply:
Hello!
Thanks for your feedback 🙌🏼 
This is solved 😄